src/main/java/com/getindata/connectors/http/internal/table/lookup/querycreators/PrefixedConfigOption.java

```diff
@@ -0,0 +1,46 @@
+package com.getindata.connectors.http.internal.table.lookup.querycreators;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.FallbackKey;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This is a ConfigOption that has an associate config option and prefix.
+ *
+ * Note that this Class used to extend ConfigOption, but at Flink 1.16, there was a new way of doing class loaders
+ * for custom content, so we could no longer extend ConfigOption.
+ */
+public class PrefixedConfigOption<T> {
```

Comment:
unfortunately this will not work as described here:
https://github.com/getindata/flink-http-connector/issues/54#issuecomment-1802402345